tests/wasm.rs

```diff
@@ -265,6 +265,24 @@ async fn get_put() {
     assert_eq!(dispatch(db, "close", "").await.unwrap(), "");
 }
 
+#[wasm_bindgen_test]
+async fn get_root() {
+    let db = &random_db();
+    assert_eq!(dispatch(db, "open", "").await.unwrap(), "");
+    assert_eq!(
```

Comment:
if it's empty do we get ""? wait -- is it even possible for it to be empty in practice? i guess not, nm.

Reply:
This test is basically testing the behavior if we call `getRoot` after `open` as callers are supposed to do. I guess it is a valid question and worth testing what happens if you call `getRoot` before open.